src/test/resources/pages/instructorFeedbackSubmitPageAwaiting.html

```diff
@@ -258,7 +258,7 @@
               </span>
             </div>
             <div class="col-sm-10">
-              <div class="panel panel-default panel-body mce-content-body content-editor empty" data-is-session-open="false" data-length-text-id="responseLength-1-1" data-recommended-text="0" id="responsetext-1-1" name="responsetext-1-1" spellcheck="false">
+              <div class="panel panel-default panel-body mce-content-body content-editor empty" contenteditable="true" data-is-session-closed="true" data-length-text-id="responseLength-1-1" data-recommended-text="0" id="responsetext-1-1" name="responsetext-1-1" spellcheck="false" style="position: relative;">
```

Comment:
@amarlearning Just a comment in passing. Am I missing something here or does this seem off? If the session is closed, then `readonly` should be set to true, meaning `contenteditable` should NOT be true. If you also think something is off can you investigate? This seems to be the case with all test files where `data-is-session-closed="true"` and since the default behaviour of readonly is to be `false`, I believe the property is not being set to `true` in the JavaScript successfully. 

One debugging measure I would suggest is to try `console.log(isSessionClosed)` after line 966 where you set the variable and see if it shows a String or a boolean.